.azure/pipelines/ci.yml

```diff
@@ -739,7 +739,7 @@ stages:
     parameters:
       platform:
         name: 'Managed'
-        container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-f39df28-20191023143754'
+        container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-20210714125435-9b5bbc2'
```

Comment:
Is this update valid❔ Should we instead move to `centos-7-source-build-20210714125450-5d87b80`❔

Reply:
@MichaelSimons @mthalman @crummel❔

Reply:
Thanks @MichaelSimons❕ I'll leave this as-is 😺